updating/understanding_updates/intro-to-updates.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Introduction to OpenShift updates
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update an {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators are automatically notified when an update is available for their cluster.
+
+The OpenShift Update Service (OSUS) builds a graph of update possibilities based on release images in the registry.
+The graph is based on recommended, tested update paths from a specific version. {product-title} clusters connect to the Red Hat Hybrid Cloud servers and identify which clusters the user is running, along with the version information. OSUS responds with information about known update targets.
```

Comment:
> {product-title} clusters connect to the Red Hat Hybrid Cloud servers and identify which clusters the user is running, along with the version information.

Hmm, is it saying managed OCP clusters? If we are going to talk about how a cluster gets the available update in a high level, we may say something like,

```
Red Hat hosts a public OpenShift Update Service (OSUS) service which serves a graph of update possibilities based on the OCP release images in the official registry. The graph contains recommended, supported but not recommended updates for any public OCP release. OpenShift Container Platform clusters are configured to connect to the OSUS service by default and OSUS responds to clusters with information about known update targets.
```

Reply:
Yeah the "Red Hat Hybrid Cloud" mention is a bit weird. The upgrade availability works differently in standalone OCP (what I described in my draft) and in managed clusters, "Hybrid Cloud" sounds like it talks about the latter. 

What Yang suggested sound correct, but I would argue against trying to be hyperprecise by talking about "recommended, supported but not recommended".

Reply:
Okay, I'll include Yang's suggested rewrite. Instead of "recommended, supported but not recommended", would it be fine to just say "The graph contains available updates for any public OCP release."?

I could choose other wording as well, such as "supported updates".